src/main/charts/agent/templates/tests/test-application-status.yaml

```diff
@@ -0,0 +1,31 @@
+{{/*TODO: Once status endpoint is available re-instate this test*/}}
```

Comment:
Bamboo agents to not have a `/status` endpoint yet. Ticket created to be actioned by Bamboo team:
https://bulldog.internal.atlassian.com/browse/BAMBK8S-84